images/macos/scripts/build/install-unxip.sh

```
@@ -0,0 +1,15 @@
+#!/bin/bash -e -o pipefail
```

Shellcheck (0.10.0)

[error] On most OS, shebangs can only specify a single parameter.

(SC2096)